app/schema.graphql

```diff
@@ -566,8 +566,8 @@ type Query {
     """HDBSCAN cluster selection epsilon"""
     clusterSelectionEpsilon: Float! = 0
   ): [Cluster!]!
-  streamingLastUpdatedAt: DateTime
-  spans(timeRange: TimeRange, traceIds: [ID!], first: Int = 50, last: Int, after: String, before: String, sort: SpanSort, rootSpansOnly: Boolean, filterCondition: String): SpanConnection!
+  streamingLastUpdatedAt(projectName: String): DateTime
```

Comment:
I think ultimately we might just nest these under the `Project` node - just a heads-up. I think this is a good intermediary step so the UI has time to adjust

Reply:
yea, backward compatibility is a goal of this PR